fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1292,13 +1294,14 @@ void determineCommittedTransactions(CommitBatchContext* self) {
 // This first pass through committed transactions deals with "metadata" effects (modifications of txnStateStore, changes
 // to storage servers' responsibilities)
 ACTOR Future<Void> applyMetadataToCommittedTransactions(CommitBatchContext* self) {
-	auto pProxyCommitData = self->pProxyCommitData;
+	state ProxyCommitData* const pProxyCommitData = self->pProxyCommitData;
+	state std::unordered_set<EncryptCipherDomainId> extraDomainIds;
```

Comment:
nit: maybe we can rename the variable `extra` prefix doesn't capture the purpose well, I don't any great suggestions, but, how about `postResolutionTenantsIds` ?

Reply:
With the inline comment you suggested, is it easier to understand?

Reply:
updated the logic, which should make the naming look better.